app/views/pages/home.html.erb

```diff
@@ -1,5 +1,13 @@
 <h1>Pages#home</h1>
 <p>Find me in app/views/pages/home.html.erb</p>
 
+<%# if current user has a project show the contributions page %>
+<% if current_user == @sponsor %>
```

Comment:
we might show all the contributions  on the projects page, but we can change this code later perhaps